rclcpp/include/rclcpp/executor.hpp

```diff
@@ -913,14 +800,14 @@ class Executor
 private:
   RCLCPP_DISABLE_COPY(Executor);
 
-  std::vector<std::weak_ptr<rclcpp::node::Node>> weak_nodes_;
-  typedef std::list<void*> SubscriberHandles;
+  std::vector<std::weak_ptr<rclcpp::node::Node> > weak_nodes_;
```

Comment:
The latest version of uncrustify (0.61) introduces this configuration option for it: `sp_permit_cpp11_shift`

I updated the config and all PRs where necessary.